command/git_push.go

```diff
@@ -96,6 +97,11 @@ func (c *gitPush) Execute(ctx context.Context, comm client.Communicator, logger
 			continue
 		}
 
+		if len(modulePatch.PatchSet.Summary) == 0 {
+			logger.Execution().Info("Skipping empty patch file...")
```

Comment:
should we put the filename that's empty here just to make it more clear (or the module name?)

Reply:
done.